spec/core_extensions_spec.rb

```diff
@@ -182,6 +182,14 @@
         end
       end
 
+      context 'when currency argument is a hash' do
+        subject(:hash) { {cents: 100, currency: {id: 'USD'}} }
+
+        it 'converts Hash to Money using id from currency hash' do
+          expect(hash.to_money).to eq(Money.new(100, 'USD'))
```

Comment:
Good eye! Forgot that USD is the default currency. I also changed the description of the test as it's `iso_code` and not `id` that is used. Amended my last commit.